packages/app/src/components/DBRowTable.tsx

```diff
@@ -699,7 +702,7 @@ function mergeSelectWithPrimaryAndPartitionKey(
     primaryKeys.trim() !== '' ? splitAndTrimCSV(primaryKeys) : [];
   const allKeys = [...partitionKeyArr, ...primaryKeyArr];
   if (typeof select === 'string') {
-    const selectSplit = splitAndTrimCSV(select);
```

Comment:
This is the root cause, make input:
`JSONExtractString(Body, 'object', 'c'),JSONExtractString(Body, 'object', 'note')`
become
`["JSONExtractString(Body", "'object'", "'c')", "'note')"]`
after dedupe set(`selectColumns`).

Reply:
I checked the `csv-parse`, however it does not work well with parentheses and brackets. 
Parsing option `quote` only works for same character such as: `"content"`.

Reply:
got it. in this case, I think we want to make sure the unit tests cover all use cases. Does it work with the select field like `"foo,bar",field2,field3` as well?

Reply:
It will split it to:
```
[
  '"foo',
  'bar"',
  'field2',
  'field3'
]
```
It should works fine if we don't get the input like: `"foo,bar","foo,differentBar",field2,field3` and also doing the dedupe.

Reply:
I will work on quote stuff, will update PR later.

Reply:
cool, I'd imagine the return value from the example above is `["foo,bar", "field2", "field3"]` although it's a corner case

Reply:
update the split function to support single quote and double quote, feel free to check test case.